models/fates/R/model2netcdf.FATES.R

```diff
@@ -11,291 +11,105 @@
-        #z <- z+1
-        #output[[z]] <- getnetCDF(nc, "lon")                # longitude, degrees east
-        #z <- z+1
-        #output[[z]] <- getnetCDF(nc, "lat")                # latitude, degrees north
-        ## !!
-        
-        # 
-        z <- z+1
-        output[[z]] <- getnetCDF(nc, "area")                # grid cell areas, [lngrid]
-        # #ncatt_get(nc, "area")
-        z <- z+1
-        output[[z]] <- getnetCDF(nc, "landfrac")            # land fraction, landfrac[lndgrid]
-        # #ncatt_get(nc, "landfrac")
-        z <- z+1
-        output[[z]] <- getnetCDF(nc, "landmask")            # land/ocean mask (0.=ocean and 1.=land), landmask[lndgrid]
-        z <- z+1
-        output[[z]] <- getnetCDF(nc, "pftmask")             # pft real/fake mask (0.=fake and 1.=real)
-        z <- z+1
-        output[[z]] <- getnetCDF(nc, "nbedrock")            # index of shallowest bedrock layer
-        #z <- z+1
-        #output[[z]] <- getnetCDF(nc, "ZSOI")                # soil depth, in meters
-        
-        # output[[16]] <- NA 
-        # output[[17]] <- NA
-        # output[[18]] <- NA
-        # output[[19]] <- NA
-        # # ... many more here still to fill in but now on to something more interesting
-        z <- z+1
-        output[[z]] <- getnetCDF(nc, "AR")                  # autotrophic respiration, [lndgrid,time]
-        # #ncatt_get(nc, "AR")
-        output[[z]] <- ifelse(output[[z]] == 1e+36, miss.val, 
-                             misc.convert(output[[z]],"umol C m-2 s-1","kg C m-2 s-1"))
-        z <- z+1
-        output[[z]] <- getnetCDF(nc, "GPP")                 #
-        # #ncatt_get(nc, "GPP") # units of GPP
-        output[[z]] <- ifelse(output[[z]] == 1e+36, miss.val, 
-                              misc.convert(output[[z]],"umol C m-2 s-1","kg C m-2 s-1"))
-        
-        # ... still more here to go!
-        
-        #******************** Declare netCDF variables ********************#
-        rm(z)
+
+        #******************** Declare netCDF dimensions ********************#
         nc_var  <- list()
+        sitelat <- ncvar_get(ncin,"lat")
+        sitelon <- ncvar_get(ncin,"lon")
         ## time variable based on internal calc, nc$dim$time is the FATES output time
         t <- ncdim_def(name = "time", units = paste0("days since ", year, "-01-01 00:00:00"),
-                       vals = time, calendar = "standard", 
+                       vals = as.vector(time), calendar = "standard", 
                        unlim = TRUE)  # a direct analog of internal FATES output dim "time"
         lat <- ncdim_def("lat", "degrees_north", vals = as.numeric(sitelat), longname = "coordinate_latitude")
-        ##!! default is degrees east except we always input as degrees west, need to make sure this is sorted for all
-        ## pecan functions!!
         lon <- ncdim_def("lon", "degrees_east", vals = as.numeric(sitelon), longname = "coordinate_longitude")
```

Comment:
@mdietze not sure if this comment was clear but I noticed that we often read in the long in -XXX.XXXX i.e. degrees west but we always save as degrees east and I am not sure we are converting properly in all of the model2netcdf codes